plugins/Satellites/src/gui/SatellitesDialog.hpp

```diff
@@ -163,7 +165,7 @@ private slots:
 #endif
 
 	Ui_satellitesDialog* ui;
-	bool satelliteModified;
+	bool satelliteModified; // currently not used
```

Comment:
You can comment this away then. Also its actual (although useless) uses in SatellitesDialog::updateSatelliteData() and in the constructor.